pontoon/tags/urls.py

```diff
@@ -0,0 +1,17 @@
+
+from django.conf.urls import url
+
+import views
+
+
+urlpatterns = [
+    # Project tag page
+    url(r'^projects/(?P<project>[\w-]+)/tags/(?P<tag>[\w-]+)/$',
+        views.ProjectTagView.as_view(),
+        name='pontoon.tags.project.tag'),
+
+    # AJAX view: Project tag
+    url(r'^projects/(?P<project>[\w-]+)/ajax/tags/(?P<tag>[\w-]+)/$',
+        views.ProjectTagView.as_view(),
+        name='pontoon.projects.ajax.tags'),
```

Comment:
Nit: Looking at the `localization` module, `pontoon.tags.ajax.teams` would be more consistent.